es6-const-let/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>es6-const-let</title>
+</head>
+<body>
+  <script>
+    const greetings = [
+      'hello!',
+      'konichiwa!',
+      'nihao',
+      'bonjour',
+      'gutten tag',
+      'ola'
+    ];
+
+    function logAllGreetings(greetingList) {
+      let i = 0;
+      while (i < greetingList.length) {
+        console.log('Current greeting:', greetingList[i]);
+        i++;
+      }
+    }
+    logAllGreetings(greetings);
+
+    function getRandomGreeting(greetingList) {
+      const randomIndex = Math.floor(Math.random() * greetingList.length);
+      let greeting = greetingList[randomIndex];
+      if (randomIndex === 1) {
+        greeting = greeting + ' is a japanese greeting.';
+      }
+      return greeting;
+    }
+    console.log('Random greeting:', getRandomGreeting(greetings));
+
+    function addToGreetings(newGreeting, greetingList) {
+      greetingList.push(newGreeting);
+    }
+    addToGreetings('namaste', greetings);
+    logAllGreetings(greetings);
+    let variable = 'meow';
```

Comment:
Is `variable` ever re-assigned? If not, `let` is probably not the best choice here:

---

![image](https://user-images.githubusercontent.com/16143771/127089265-e833515a-16c3-4d41-b73a-541b31660c03.png)

Reply:
![Screen Shot 2021-07-26 at 9 43 03 PM](https://user-images.githubusercontent.com/24905313/127092003-42860768-ce8b-4ac6-811e-da44c3297372.png)